whois-api/src/test/java/net/ripe/db/whois/api/rdap/RdapObjectMapperTest.java

```diff
@@ -1,798 +0,0 @@
-package net.ripe.db.whois.api.rdap;
```

Comment:
why have you removed this test?

Reply:
Yes, its completely obsolete, I confirmed all the tests were moved to integration tests.